lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -2612,6 +2628,31 @@ def _add_to_dict(metadata):
         return d
 
 
+def loggly_time_string(dt):
+    r = dt.strftime("%Y-%m-%d") + "T" + dt.strftime("%H") + "%3A" + dt.strftime("%M") + "%3A"
```

Comment:
you can mix in constants with strftime: `%Y-%m-%dT%H%M%SZ`

Also consider using https://docs.python.org/2/library/urllib.html#urllib.urlencode instead of manually doing `%3A`